### ISSUE TITLE
Any directions for offering contributions?

### ISSUE THREAD

**Issue by jquense** [FormidableLabs/radium-bootstrap#17]
If one wanted to help with getting a feature or module implemented are their any documented guidelines or suggestions on where to start? Are their any particular priorities for things (navbar before modal, etc).

How about features that are more function than style, like Affix, or scroll spy. Is the goal to create Radium compatible examples of all of bootstrap or only the bits that are really css heavy (like the navbar).

very cool project!

**Comment by maur8ino** [comment]
+1!

**Comment by alexlande** [comment]
Hey! Thanks for the interest. We unfortunately don't have any documentation or contribution guidelines, but I suppose I get should get to work on that.

We don't have a particular stack ranking of components to make at this point, but the hope is to translate all of Bootstrap, including the functional portions.

If you have a particular component you'd like to start working on, please open an issue to track it and go for it! If you have any questions, any of the contributors can help out. To my knowledge, @kenwheeler is working on (or thinking about working on) navbars, and @rgerstenberger is working/planning to build dropdowns.

Other than that, any component is fair game. I'll work on writing some docs, contributor guidelines, roadmap, etc to help clarify this stuff for the future.

**Comment by chrisvfritz** [comment]
Is this still "a work in progress" as described in the README? I really like the idea of this project, but I see it hasn't been updated in 4 months. I'm hesitant to make contributions if my small part won't add up to a greater whole.

**Comment by alexlande** [comment]
@chrisvfritz: Yeah, this never got off the ground, unfortunately. I still think it's a great idea and would really like to see it happen, but I think it's pretty clear that no one on the Radium team has the time to build and maintain it. We'll probably close this repo down in the next week or so.

**Comment by chrisvfritz** [comment]
@alexlande Thanks for the update! I'm still loving Radium and really appreciate all the hard work you folks are putting into it. :smiley:
